rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -16,22 +13,24 @@
     WebSocketDisconnect,
 )
 from fastapi.middleware.cors import CORSMiddleware
+from fastapi.responses import JSONResponse
+from fastapi.staticfiles import StaticFiles
 
-from autogpt_server.data import db, execution, block
-from autogpt_server.data.graph import (
-    create_graph,
-    get_graph,
-    get_graph_ids,
-    Graph,
-)
+import autogpt_server.data.graph
+import autogpt_server.server.ws_api
+from autogpt_server.data import block, db, execution
```

Comment:
Change `autogpt_server.data.graph.*` annotations to `graph_db.*` for readability
```suggestion
import autogpt_server.server.ws_api
from autogpt_server.data import block, db, execution, graph as graph_db
```